client/app/bundles/course/video/submission/containers/VideoPlayer.jsx

```diff
@@ -62,12 +65,24 @@ class VideoPlayer extends React.Component {
     });
   }
 
+  componentDidMount() {
+    if (this.props.onTick) {
+      this.timer = setInterval(this.props.onTick, tickMilliseconds);
+    }
+  }
+
   componentWillReceiveProps(nextProps) {
     if (nextProps.forceSeek) {
       this.player.seekTo(nextProps.playerProgress);
     }
   }
 
+  componentWillUnmount() {
```

Comment:
Just a random thought: would it be a good idea to try to send events before the component unmounts? (Probably when the browser winder closes)

Reply:
That makes sense. I'll add another call to `onTick`